functional_tests/scylla_operator/libs/helpers.py

```diff
@@ -58,6 +59,21 @@ def wait_for_scylla_operator_rollout_complete(db_cluster: ScyllaPodCluster, time
     return status
 
 
+def wait_for_resource_absence(db_cluster: ScyllaPodCluster,
+                              resource_type: str, resource_name: str,
+                              step: int = 2, timeout: int = 60):
```

Comment:
Please, add a type hint for the return value (`None`, I guess)

Reply:
Done